packages/@ama-sdk/schematics/collection.json

```diff
@@ -31,6 +31,10 @@
       "description": "Generate a Java SDK source code based on swagger specification",
       "factory": "./schematics/java/client-core/index#ngGenerateJavaClientCore",
       "schema": "./schematics/java/client-core/schema.json"
+    },
+    "update": {
```

Comment:
As we are not in the `migration.json` I would expect a name that explain what it does